emoji/src/main/java/com/vanniktech/emoji/Utils.java

```diff
@@ -71,29 +61,33 @@ static boolean shouldOverrideRegularCondition(@NonNull final Activity context, f
     return false;
   }
 
-  static int getInputMethodHeight(final Context context, final View rootView) {
+  @SuppressWarnings({"unchecked", "JavaReflectionMemberAccess"}) static int getInputMethodHeight(final Context context, final View rootView) {
```

Comment:
@vanniktech I really dislike the "put annotations on the same line" rule, since it leads to awful things like this. Do we really want to enforce that?

Reply:
Yeah we should not enforce it.